Gemfile

```diff
@@ -12,14 +12,12 @@ source "https://rubygems.org"
 gem "chef", path: "."
 
 gem "chef-config", path: File.expand_path("../chef-config", __FILE__) if File.exist?(File.expand_path("../chef-config", __FILE__))
-gem "rake"
 gem "bundler"
 gem "cheffish" # required for rspec tests
 
 group(:omnibus_package) do
   gem "appbundler"
   gem "rb-readline"
-  gem "nokogiri", "~> 1.7.2"
```

Comment:
I forget the exact message, but I needed to bump nokogiri. Now that I think about it, it might have been related to 2.4 so maybe I could back this out. I'm not sure which is better. But yeah, I copied the pattern from your big change in master.